compliance_checker/ioos.py

```diff
@@ -740,10 +743,39 @@ def check_geophysical_vars_have_attrs(self, ds):
         list: list of Result objects
         """
 
-        return self._check_vars_have_attrs(
-            ds, get_geophysical_variables(ds), self.geophys_check_var_attrs
+        # get geophysical variables
+        geophys_vars = get_geophysical_variables(ds)  # list of str
+        results = self._check_vars_have_attrs(  # list
+            ds, geophys_vars, self.geophys_check_var_attrs
         )
 
+        # accuracy, precision, resolution must be numeric
+        # https://github.com/ioos/compliance-checker/issues/839
+        for v in get_geophysical_variables(ds):
+            _v = ds.variables[v]
+            for att in ("accuracy", "precision", "resolution"):
+                val = getattr(_v, att, None)
+                if isinstance(val, Number):
+                    r = True
+                else:
+                    r = False
+
+                results.append(
+                    Result(
+                        BaseCheck.MEDIUM,
+                        r,
+                        f"geophysical_variable:{att}",
```

Comment:
Use `str.format()` instead of f-strings for backwards compatibility with older Python 3 versions.

Reply:
f-strings are new on Python 3.6 and, with most [scientific software adopting NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html), that Python is no longer supported. That bares the question should we support Python <=3.6?

Reply:
Hmmm.  Not that long ago we dropped Python 2.  I wasn't aware of this NEP.  My NumPy version appears to be higher than the Python 3.6 supported version, so it's likely safe.  I don't think we've explicitly pinned a NumPy version, but rather it's going to be dependent on some of the other packages such as NetCDF.  To be on the safe side, I'd say convert to `str.format` for now, with a view towards supporting Python 3.6 as a minimum baseline in the near future.

Reply:
Going to target 3.6+ from now on, so this is OK.